presentation/src/main/java/com/architect/coders/mu8/utils/BindingAdapters.kt

```diff
@@ -0,0 +1,11 @@
+package com.architect.coders.mu8.utils
+
+import android.view.View
+import androidx.databinding.BindingAdapter
+
+@BindingAdapter("visible")
+fun View.setVisible(visible: Boolean?) {
```

Comment:
Same comment about the functions, can we avoid it? I'll double check Antonio's video tho.